todo-app/src/component/Todos.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { observer, inject } from "mobx-react";
+
+// Component
+import TodoList from './TodoList';
+import AddTodoForm from './AddTodoForm';
+import Footer from './Footer';
+
+@inject('TodoStore')
+@observer
+export default class Todos extends React.Component {
+
+  render() {
+    const { handelCheck,
+      handelDeleteItem,
+      handleDateChange,
+      handleDescriptionChange,
+      handleSubmit,
+      newDeadline,
+      newDescription,
+      todos,
+    } = this.props.TodoStore;
+
+    const hasTodos = todos.length > 0;
+    
+    return (
+      <div className="todosContainer">
+        <h3>Todo-List</h3>
+        <AddTodoForm description={newDescription}
```

Comment:
So you are connecting your `Todos` component to the store, but then passing a lot of things as props instead of connecting the child components directly. Both options work, but I wonder if you had a reason for this?

The main point of using a `Provider` is that we can connect child components anywhere in the tree to the data they need, even if it's just one field from the store.